mappings/net/minecraft/fluid/FlowingFluid.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
+CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowingFluid
```

Comment:
flowing or flowable? so is this like flowing water/lava excluding water/lava source blocks?

Reply:
Well rather it's a fluid that __*could*__ flow, this includes the source blocks also

Reply:
Then probably `FlowableFluid`, flowing sounds like it only includes flowing water/lava